src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -120,22 +120,6 @@ <h4 ng-if="$ctrl.useV3 === 'true'" class="panel-title  border-bottom-small  visi
           </div>
         </div>
       </div>
-      
-        <div ng-if="$ctrl.useV3 === 'true'" class="row">
-          <div class="col-sm-6">
```

Comment:
In the other instance, you left this div in place. Was the intention to put the `col-sm-12` here instead of in the component template?